src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -26,22 +29,33 @@ protected AbstractEditorTest(VisualStudioInstanceFactory instanceFactory, string
             string projectTemplate)
            : base(instanceFactory)
         {
-            VisualStudio.SolutionExplorer.CreateSolution(solutionName);
-            VisualStudio.SolutionExplorer.AddProject(new ProjectUtils.Project(ProjectName), projectTemplate, LanguageName);
+            _solutionName = solutionName;
+            _projectTemplate = projectTemplate;
+        }
 
-            // Winforms and XAML do not open text files on creation
-            // so these editor tasks will not work if that is the project template being used.
-            if (projectTemplate != WellKnownProjectTemplates.WinFormsApplication &&
-                projectTemplate != WellKnownProjectTemplates.WpfApplication &&
-                projectTemplate != WellKnownProjectTemplates.CSharpNetCoreClassLibrary)
+        protected abstract string LanguageName { get; }
+
+        public override async Task InitializeAsync()
+        {
+            await base.InitializeAsync();
```

Comment:
Can you add an explicit ConfigureAwait here? We don't have the analyzer turned on for test projects, but I'd prefer if we're explicit about our expectations in support code

Reply:
➡️ Updated all 10 cases to use `ConfigureAwait(true)` (no other changes)